src/styles/common.scss

```diff
@@ -236,6 +236,36 @@
     margin-bottom: 1%;
 }
 
+@media (max-width: $desktop-width) {
+    .textBanner {
+        padding: 70px 14%;
+    }
+
+    .textBannerSection {
+        margin-left: 2vw;
+        margin-right: 2vw;
+    }
+
+    .textBannerSubtitle {
+        font-size: 1.8vw;
+        margin-bottom: 2%;
+    }
+}
+
+@media (max-width: $tablet-width) {
+    .textBanner {
+        padding: 70px 0;
+    }
+
+    .textBannerSubtitle {
+        font-size: 14px;
+    }
+
+    .textBannerTitle {
+        font-size: 6.3vw;
+    }
```

Comment:
The min size needs to be limited because it goes to the same size as subtitle as you shrink the window.